<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analy</class>
 <widget class="QMainWindow" name="Analy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QPushButton" name="pbt3">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>时间区间</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>整体</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pbt2">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>时间区间</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pbt1">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>特定时间</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>数据展示功能分区</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pbt4">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>个人</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
